sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -402,9 +403,49 @@ export class StreamingReceiver extends MessageReceiver {
     await this._receiverHelper.suspend();
   }
 
-  async init(useNewName: boolean, abortSignal?: AbortSignalLike): Promise<void> {
-    const options = this._createReceiverOptions(useNewName, this._getHandlers());
-    await this._init(options, abortSignal);
+  async init(
+    // TODO: make onError? required
```

Comment:
Seems appropriate to make it required

Reply:
Yes, definitely. Was a stopgap and is now removed :)